### ISSUE TITLE
Security Audit

### ISSUE THREAD

**Issue by MarioArnt** [MarioArnt/bringgle#11]
Need to audit the security of the app.
We purposely don't want the user to have to log in. 
He can create and join list as a guest user. That's fine but we don't use CRSF tokens.
Nee to fix that vulnerability.

**Comment by mohammadobaid1** [comment]
Will referrer based protection works for you to prevent csrf attacks ?

**Comment by MarioArnt** [comment]
If you mean implement referrer policy based on CORS headers it would be great.
This article seems quite exhaustive on the topic https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)_Prevention_Cheat_Sheet#General_Recommendations_For_Automated_CSRF_Defense

**Comment by mohammadobaid1** [comment]
Yeah referer based protection . Wil have to do at api level which checks for referer header value before process something . I will try to do it in coming weekend and then submit PR once completed. :)

**Comment by MarioArnt** [comment]
Great, thanks a lot !
